Formula/p/parsedmarc.rb

```diff
@@ -324,11 +325,6 @@ class Parsedmarc < Formula
     sha256 "4f50ed74ab84d474ce614475e0b8d5047ff080810aac5d01ea25231cfc944b0c"
   end
 
-  resource "setuptools" do
```

Comment:
Should we keep this?

Reply:
If it works, I don't think we need to keep it